docs/index.html

```diff
@@ -27,6 +27,9 @@
             padding: 0;
             box-sizing: border-box;
         }
+
+        ::-webkit-datetime-edit-fields-wrapper { background: transparent; }
```

Comment:
What's this?

Reply:
Oh. I was trying out ways to customize the dropdowns and I forgot to remove them when I made the commit. I'll remove those in a moment